packages-ts/gauntlet-starknet-cli/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@chainlink/gauntlet-starknet-cli",
+  "name": "@chainlink/starknet-gauntlet-cli",
```

Comment:
The package names changed, but the directories didn't which seems to confuse my LSP.